task7_2/task7_2/Program.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace task7_2
+{
+    internal class Program
+    {
+        static void Main(string[] args)//задание 7.2
+        {
+            Console.WriteLine("Введите координаты точки: х и у");
+            double x = double.Parse(Console.ReadLine());
+            double y = double.Parse(Console.ReadLine());
+            Console.WriteLine(IsPointInArea(ref x, ref y));
+            Console.ReadKey();
+
+        }
+        static bool IsPointInArea(ref double x, ref double y)
+        {
+            if (y >= 1 | y<= -3)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
```

Comment:
Без особых причин следует избегать передачи данных, имеющих тип-значение, по ссылке.
Методы в классе лучше отделять пустой строкой. Так код читается легче.
Используйте ленивую дизъюнкцию!
В методе можно было обойтись одной строкой.
```suggestion
            Console.WriteLine(IsPointInArea(x, y));
            Console.ReadKey();

        }
        
        static bool IsPointInArea(double x, double y)
        {
            return y >= 1 || y <= -3;
        }
```